DownloadManager/Plugin.DownloadManager.Android/DownloadManagerImplementation.cs

```diff
@@ -165,7 +166,11 @@ public void UpdateFileProperties (ICursor cursor, DownloadFileImplementation dow
 
             switch ((DownloadStatus)cursor.GetInt (cursor.GetColumnIndex (Android.App.DownloadManager.ColumnStatus))) {
             case DownloadStatus.Successful:
-                downloadFile.StatusDetails = default(string);
+                if (string.IsNullOrEmpty(downloadFile.DestinationPathName))
```

Comment:
Good catch! 😅

Reply:
I've found I can also do
`downloadFile.DestinationPathName = cursor.GetString(cursor.GetColumnIndex("local_uri"));`
instead of the if statement and line 70.

I get a `file://` uri back when using a custom location, `content://` for temp locations. What do you think?

Reply:
Oh and your comment had to be in the Mac file, didn't it? :P